packages/@uppy/utils/src/Translator.ts

```diff
@@ -1,7 +1,21 @@
-import has from './hasProperty.js'
+import has from './hasProperty.ts'
 
-function insertReplacement (source, rx, replacement) {
-  const newParts = []
+export interface Locale<T extends number = number> {
+  strings: Record<string, string | Record<T, string>>
+  pluralize: (n: number) => T
+}
+
+type Options = {
+  smart_count?: number
+  [key: string]: string | number | undefined
+}
+
+function insertReplacement(
+  source: Array<string | unknown>,
+  rx: RegExp,
+  replacement: string | number,
+): Array<string | unknown> {
+  const newParts: Array<string | unknown> = []
```

Comment:
We have a lot of `unknown` here. Can we get rid of it?

Reply:
https://github.com/transloadit/uppy/blob/dca3777b9a80460957794680904582523428355a/packages/@uppy/utils/src/Translator.ts#L21-L24

What you would put instead of `unknown`?